packages/playground/blueprints/src/lib/steps/apply-wordpress-patches/wp-content/mu-plugins/0-playground.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Add a notice to wp-login.php offering the username and password.
+ */
+add_action(
+	'login_message',
+	function () {
+		return <<<EOT
+<div class="message info">
+	<strong>username:</strong> <code>admin</code><br /><strong>password</strong>: <code>password</code>
```

Comment:
@adamziel Should this be configurable? If we're demoing a plugin that allows users to create different permissions/roles for different users, a plugin vendor might want to list multiple accounts here.

Reply:
Eventually, yes! For now, hardcoded credentials seem to do the job – there haven't been many requests related to iterating on this API.